lib/src/fileset.pest

```diff
@@ -89,3 +89,5 @@ program_or_bare_string = _{
         | bare_string_pattern ~ EOI
         | bare_string ~ EOI )
 }
+
+alias_declaration = _{ SOI ~ identifier ~ EOI }
```

Comment:
It's probably better to use `strict_identifier` here, because `identifier` rule in fileset is lax.

We can also add function aliases, but that can be implemented by a separate patch.

Reply:
> `strict_identifier`
Makes sense, implemented